modules/nf-core/cnaqc/tests/nextflow.config

```diff
@@ -0,0 +1,5 @@
+process {
+  withName: 'CNAQC' {
+    ext.args = { "--genome GRCh38 --matching_strategy rightmost --purity_error 0.05 --muts_per_karyotype 25 --min_absolute_karyotype_mutations 100" }
```

Comment:
This method of using args is fine for now but newer Nextflow syntax changes will deprecate this. (just something to keep in mind in the future)